gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -444,10 +444,29 @@ def test_end_info_key(self):
     variant_2.calls.append(VariantCall(name='Sample1', genotype=[0, 1]))
     variant_2.calls.append(VariantCall(name='Sample2', genotype=[1, 1]))
     read_data = self._create_temp_file_and_read_records(
-        [phaseset_header_line] + _SAMPLE_HEADER_LINES[1:] + record_lines)
+        [end_info_header_line] + _SAMPLE_HEADER_LINES[1:] + record_lines)
     self.assertEqual(2, len(read_data))
     self._assert_variants_equal([variant_1, variant_2], read_data)
 
+  def test_end_info_key_unknown_number(self):
+    end_info_header_line = (
+        '##INFO=<ID=END,Number=.,Type=Integer,Description="End of record.">\n')
+    record_lines = ['19	123	.	A	.	.	.	END=1111	GT	1/0	0/1\n']
+    variant_1 = Variant(
+        reference_name='19', start=122, end=1111, reference_bases='A')
+    variant_1.calls.append(VariantCall(name='Sample1', genotype=[1, 0]))
+    variant_1.calls.append(VariantCall(name='Sample2', genotype=[0, 1]))
+    read_data = self._create_temp_file_and_read_records(
+        [end_info_header_line] + _SAMPLE_HEADER_LINES[1:] + record_lines)
+    self.assertEqual(1, len(read_data))
+    self._assert_variants_equal([variant_1], read_data)
+
+    # Test with invalid number of fields in END (should only have 1 number).
+    record_lines.append('19	124	.	A	.	.	.	END=11,12	GT	1/0	0/1\n')
+    with self.assertRaises(ValueError):
+      self._create_temp_file_and_read_records(
+          [end_info_header_line] + _SAMPLE_HEADER_LINES[1:] + record_lines)
```

Comment:
I think we should break this into two separate cases so each case tests just one thing.

Reply:
Done.